dune-project

```diff
@@ -0,0 +1,3 @@
+(lang dune 1.11)
+(name indexmap)
+(explicit_js_mode)
```

Comment:
Same question as in lwt-binio - is this necessary?

Reply:
same reply :)